packages/portfolio-contract/src/type-guards.ts

```diff
@@ -253,11 +261,14 @@ export const PortfolioStatusShapeExt: TypedPattern<StatusFor['portfolio']> =
       positionKeys: M.arrayOf(PoolKeyShapeExt),
       flowCount: M.number(),
       accountIdByChain: M.recordOf(
-        M.or('agoric', 'noble'), // ChainAccountKey
-        M.string(), // AccountId
+        ChainNameExtShape,
+        M.string(), // XXX no runtime validation of AccountId
       ),
     },
-    { targetAllocation: TargetAllocationShapeExt },
+    {
+      depositAddress: M.string(), // XXX no runtime validation of Bech32Address
```

Comment:
I think we have an issue for string pattern matches but I can't find it on my phone